build.sbt

```diff
@@ -33,6 +33,7 @@ libraryDependencies ++= {
   val scalaTestV  = "3.0.1"
   val catsV = "0.9.0"
   val lenthallV = "0.24-c8af383-SNAP"
+  val cromwellV = "27-091ed3b-SNAP"
```

Comment:
When we merge to develop, artifactory gets a new JAR for that project, with the github hash and SNAP attached to the name.

(When we do a real release, it gets a jar named without the hash or the SNAP)